src/compiler/scala/tools/nsc/reporters/ConsoleReporter.scala

```diff
@@ -79,7 +80,7 @@ class ConsoleReporter(val settings: Settings, reader: BufferedReader, writer: Pr
       reader.read match {
         case 'a' | 'A' =>
           new Throwable().printStackTrace(writer)
-          System.exit(1)
+          throw SystemExit(1)
```

Comment:
I don't know if this feature is ever used, but if it were necessary to avoid exit for some reason, and you used the trick to minimize the output to `aborted`, does throwing here work? Not sure this has a partest to confirm.

Reply:
I'm not worried about this. I doubt it's used much.

Reply:
I just tried this out (for refactored reporting), and you use s for stack and a to quit looking at errors, so maybe whatever it does is a feature.